classes/PodsUI.php

```diff
@@ -3030,7 +3068,18 @@ public function manage( $reorder = false ) {
 										<?php esc_html_e( $filter_field['label'] ); ?>
 									</label>
 								<?php
-									echo PodsForm::field( 'filter_' . $filter, $value, 'pick', $options );
+									// Prevent p div issues.
+									echo str_replace(
+										array(
+											'<div',
+											'</div>',
+										),
+										array(
+											'<span',
+											'</span>',
+										),
+										PodsForm::field( 'filter_' . $filter, $value, 'pick', $options )
+									);
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 0f2606a7821d8f4215ca1e6c9cf6e42d</sub>